modules/sts-mode-installing-manual-run-installer.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="sts-mode-installing-manual-run-installer_{context}"]
+= Running the installer
+
+. Run the {product-title} installer:
```

Comment:
```suggestion
* Run the {product-title} installer:
```

Reply:
oh! yep didn't even think about that now that this is in its own file :D